13_message_delivered/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Space+Mono&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Secret Door</title>
+  </head>
+  <body>
+    <div class="container">
+      <h1>Message you would like to pass</h1>
+      <input class="input" type="text" id="msg" />
+      <button class="button" type="submit" id="btn" onkeypress="shiftKey()">
```

Comment:
What were you trying to accomplish here?

This is not how you should add events to a button. You should add an event listener as you did at `app.js`

Reply:
its not needed I think was was trying a lot of different things , thats why it stayed there